install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -73,8 +74,10 @@ spec:
               endpoint:
                 properties:
                   external:
+                    description: The endpoint external to the cluster within which the Broker is running, which the Meshery Server will connect to
```

Comment:
The Broker isn't running "within" and endpoint, but is "exposed on".

Reply:
Done.